.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -9,8 +9,8 @@ name: Llama Benchmarking Tests
 on:
   workflow_dispatch:
   schedule:
-    # Weekdays at 4:00 AM UTC = 9:00 PM PST.
-    - cron: "0 4 * * 1-5"
+    # Weekdays at 09:50 AM UTC = 01:50 AM PST / 02:50 AM PDT
+    - cron: "50 9 * * 1-5"
```

Comment:
You might want to give this a bit of extra buffer, in case the IREE release builds take longer over time as more code, dependencies, and configurations are added.

Looking through the history at https://github.com/iree-org/iree/actions/workflows/validate_and_publish_release.yml, publication time is pretty consistently between 1:20AM and 1:30AM but I think giving a 1 hour buffer would be safer.

Reply:
There are also ways to trigger on events like releases being published, but those get harder across organizations and repositories.

Reply:
Good point, I can make it 3 AM PST for now, till we have a way to trigger this automatically after every sharktank/shortfin/IREE nightly release.